CHANGELOG.md

```diff
@@ -4,9 +4,16 @@ Notable changes since version 42.0.0, read the complete [History of Changes](htt
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+### Changed
+
+### Added
+
+### Fixed
+
+## [42.2.19] (2021-02-15)
 
 **Notable Changes**
-- Now the driver use SASLprep normalization for SCRAM authentication fixing some issues with spaces in passwords.
+- Now the driver uses SASLprep normalization for SCRAM authentication fixing some issues with spaces in passwords.
```

Comment:
I would rephrase it like "Passwords with non-ASCII spaces work now with SCRAM authentication (driver uses SASLprep normalization)" or something like that.

Reply:
Well, both issues are with passwords *with* (ASCII) spaces... so maybe:

"Passwords with spaces (ASCII and non-ASCII) now work with SCRAM authentication (driver now uses SASLprep normalization)" or something like that.

Reply:
Thanks, this indeed looks easier to understand for those who are not much into SASLprep :)

Reply:
Yes the "what" of the change is more important than than the "how" so it should come first. Switching up @jorsol's text a bit, how about this (note that "whitespace" vs "space" is intentional as "space" is just that one character):

> Passwords with ASCII and non-ASCII whitespace can be used with SCRAM authentication as the driver now performs SASLprep normalization.